contracts/core/SettingsManager.sol

```diff
@@ -134,6 +141,7 @@ contract SettingsManager is ISettingsManager, Ownable, Constants {
         pauseForexForCloseTime = _enable;
         emit EnableForexMarket(_enable);
     }
```

Comment:
just to mention that is EnableForexMarket is misleading, this enable is just pausing the market